_posts/elements/2015-05-12-forms.md

```diff
@@ -54,8 +54,8 @@ info: Forms allow users to interact with the interface, set preferences, and ent
             </div>
             <div class="field">
                 <label class="field-label field-required" for="email_address_default_suc_helper">Email address (default, has success)</label>
-                <input class="field-input input-text has-success" type="email" id="email_address_default_suc_helper" name="email_address_default_suc_helper" placeholder="e.g. jane@email.com" value="jane@domain.com" aria-describedby="email_address_default_suc_helper" required>
-                <div class="field-hint" id="email_address_default_suc_helper">
+                <input class="field-input input-text has-success" type="email" id="email_address_default_suc_helper" name="email_address_default_suc_helper" placeholder="e.g. jane@email.com" value="jane@domain.com" aria-describedby="email_address_default_suc_helper_text" required>
```

Comment:
See [my notes above](https://github.com/edx/ux-pattern-library/pull/126/files#r35094989) and please address.